PowerEditor/src/DarkMode/IatHook.h

```diff
@@ -30,7 +30,7 @@ PIMAGE_THUNK_DATA FindAddressByName(void *moduleBase, PIMAGE_THUNK_DATA impName,
 			continue;
 
 		auto import = RVA2VA<PIMAGE_IMPORT_BY_NAME>(moduleBase, impName->u1.AddressOfData);
-		if (strcmp(import->Name, funcName) != 0)
+		if (strcmp(reinterpret_cast<const char*>(import->Name), funcName) != 0)
```

Comment:
Please keep this one unchanged. The type of `Name` is a `char` but not `char *`.

Reply:
Done.
The type of `import->Name` is `CHAR`. For some reason, it is defined as `BYTE` in GCC which is `unsigned char` hence the warning.

Reply:
Oh, turns out, the latest MinGW header has defined it right:
https://github.com/mirror/mingw-w64/blob/master/mingw-w64-headers/include/winnt.h#L7226
So this is not needed anymore.